aws/resource_aws_ecs_service.go

```diff
@@ -274,6 +274,17 @@ func resourceAwsEcsService() *schema.Resource {
 				},
 			},
 
+			"propagate_tags": {
+				Type:     schema.TypeString,
+				Optional: true,
+				DiffSuppressFunc: func(k, old, new string, d *schema.ResourceData) bool {
+					if old == "NONE" && new == "" {
```

Comment:
I put this here because AWS sets this to `NONE` if the value isn't set and it's not part of the list of valid values.
Otherwise I get this error:
```
--- FAIL: TestAccAWSEcsService_withAlb (240.66s)
    testing.go:538: Step 0 error: After applying this step, the plan was not empty:

        DIFF:

        UPDATE: aws_ecs_service.with_alb
          propagate_tags: "NONE" => ""
```